cookbooks/docker/recipes/default.rb

```diff
@@ -39,7 +33,7 @@
 
 # Get Docker Compose and install it
 execute "get-docker-compose" do
-  command 'curl -L https://github.com/docker/compose/releases/download/1.10.0/docker-compose-' << `uname -s`.chomp << '-' << `uname -m`.chomp << ' -o /usr/local/bin/docker-compose'
+  command 'curl -L https://github.com/docker/compose/releases/download/1.10.0/docker-compose-Linux-x86_64 -o /usr/local/bin/docker-compose'
```

Comment:
In this case, the replaced parts are always the same, since we're using 64-bit Linux.
Concerning the rest of the URI, I shall put the version (here `1.10.0`) as a separate attribute. Thanks.